src/lib/rpc.ts

```diff
@@ -15,7 +21,10 @@ export const networkConfigUrl = (agoricNetName: string) => {
     return `https://${agoricNetName}.agoric.net/network-config`;
   }
 };
-export const rpcUrl = agoricNetSubdomain =>
+
+export const marshal = makeImportContext().fromBoard;
```

Comment:
This verb implies a function but this is a Marshaller that has functions. Please rename.

Reply:
I think it's consistent with @endo/marshal:

https://github.com/endojs/endo/tree/master/packages/marshal#usage

You call "makeMarshal" and it gives you an object with `serialize` and `unserialize`. Why isn't it `makeMarshaller`?